src/components/toolbar/index.js

```diff
@@ -25,6 +25,33 @@ class Toolbar extends Component {
     super(props)
   }
 
+  handleShortcuts = event => {
```

Comment:
We should check first if user is typing so we don't affect usability

Reply:
See https://github.com/lbryio/lbry-desktop/blob/master/src/ui/util/detect-typing.js

Reply:
I added a check to see whether the reader is selected. The shortcuts will now only work if the reader.
However I am facing some problem since some keys are registered with openseadragon, for example 'f', which flips the image. I'll see if I can find any workaround.

Reply:
Anyway don't worry about it, we can open another issue for that if you don't have the time.

Reply:
I couldn't find a way without disabling the openseadragon shortcuts sorry.